### PR TITLE
STYLE: Remove initial `<<` insertion from `itkExceptionMacro(<< "` calls

### PR DESCRIPTION
**Pull request by N-Dekker**
Let the C++ preprocessor append the specified literal string to the error message, instead of doing the equivalent `<<` insertion at run-time.

Aims to shorten the code a little bit, and improve consistency. The code base already had more than 750 `itkExceptionMacro` calls that passed a literal string directly, without having `<<` at its left side.

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

File: Modules/IO/GDCM/src/itkGDCMImageIO.cxx [hunk 1]
```diff
@@ -1211,14 +1211,14 @@ GDCMImageIO::Write(const void * buffer)
       outpixeltype.SetPixelRepresentation(static_cast<unsigned short>(std::stoi(pixelRep.c_str())));
       if (this->GetNumberOfComponents() != 1)
       {
-        itkExceptionMacro(<< "Sorry Dave I can't do that");
+        itkExceptionMacro("Sorry Dave I can't do that");
```
Comment (issakomi) [inline Modules/IO/GDCM/src/itkGDCMImageIO.cxx:1214]: “I'm sorry, Dave. I'm afraid I can't do that,” said HAL 9000
(c) 2001: A Space Odyssey

Comment (N-Dekker) [reply Modules/IO/GDCM/src/itkGDCMImageIO.cxx:1214]: Thanks for the reference, Mihail! Honestly I doubt if such an error message is very helpful to the user. In general, I'm not a fan of "funny" error messages. Unless they really help to communicate to the user what's going wrong. But anyway, that's beyond the scope of this pull request!

Comment (dzenanz) [reply Modules/IO/GDCM/src/itkGDCMImageIO.cxx:1214]: Maybe it is an error message which is not supposed to be reachable by the GDCM's users?

Comment (N-Dekker) [reply Modules/IO/GDCM/src/itkGDCMImageIO.cxx:1214]: Maybe... but then I guess the message only for someone named Dave 😸  Thanks for merging @dzenanz !

**Comment by N-Dekker** [comment]
@N-Dekker 
> Aims to shorten the code a little bit

Oops, I see now:

    +1,418 −1,394

Meaning that the PR has actually _added_ more lines of code! Oops! I'm sorry  😸 !

**Pull request merged**